src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -411,8 +415,11 @@ def _get_credentials(cmd,  # pylint: disable=too-many-statements
         login_server_suffix = get_login_server_suffix(cli_ctx)
         if not login_server_suffix:
             raise
-        login_server = '{}{}{}'.format(
-            registry_name, '-{}'.format(tenant_suffix) if tenant_suffix else '', login_server_suffix).lower()
+        login_server = '{}{}{}{}'.format(
```

Comment:
login_server should always be fetched from the registry.login_server parameter. Let us not try to construct it. We can raise an exception and not proceed further if login_server is None